src/validator/Tests.java

```diff
@@ -0,0 +1,55 @@
+package validator;
+
+import static org.junit.Assert.*;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+
+import org.junit.Test;
+
+public class Tests {
+	private static CreditCardVendorsReader reader = new CreditCardVendorsReader();
+	private static ArrayList<CreditCardVendor> vendors = reader.returnVendorArray();
```

Comment:
The goal of the test was to check the CreditCardValidator in isolation.
These 2 fields re-use the production code to initialize the test parameter - vendor.
Additionally there is only 1 vendor case tested.

I would recommend to make it independent from production code.
and prepare list of possible positive/negative cases 
and then test several positive or negative cases using @ParameterizedTest (Junit 5 annotation)
with one method - e.g. shouldReturnTrueForCorrectNumber